### PR TITLE
OAP-38: Add /ngrams endpoint to API

### PR DESCRIPTION
**Pull request by peterrauscher**
Adds the following endpoint to the API:
`/api/{handle}/ngrams`

Also removes redundant .js clean and seed tasks, handles URL-escaping of handles, and updates README to reflect changes.

### CODE REVIEW

**Review by justinoboyle — APPROVED** [review]

File: api/db/data.js [hunk 1]
```diff
@@ -13,6 +13,17 @@ async function querySuggestions(id) {
   });
 }
 
+async function queryNgrams(id) {
+  await validate.checkHandle(id);
+
+  const query = new PQ({ text: "SELECT * FROM oapen_suggestions.ngrams WHERE handle = $1", values: [id] });
```
Comment (j-sofia) [inline api/db/data.js:19]: querySuggestions function needs handle instead of item_id

File: api/routes.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const validate = require("./validate");
 
 const data = require("./db/data.js");
 
-//GET endpoint
+//GET endpoint for suggestions
 router.get("/:handle", async (req, res) => {
```
Comment (j-sofia) [inline api/routes.js:8]: Can use regex to avoid encoding the slash:
"/:handle([0-9]+.[0-9]+.[0-9]+/[0-9]+)"

File: api/routes.js [hunk 3]
```diff
@@ -13,14 +13,32 @@ router.get("/:handle", async (req, res) => {
     let responseData = await data.querySuggestions(handle);
 
     if (responseData.error && responseData.error.name === pgp.errors.QueryResultError.name) {
```
Comment (j-sofia) [inline api/routes.js:15]: need const pgp = require("pg-promise"); at the top of the file

**Review by j-sofia — CHANGES_REQUESTED** [review]

**Comment by peterrauscher** [comment]
Resolves OAP-38 and resolves OAP-42

**Pull request merged**